### ISSUE TITLE
Gradle build doesn't bundle realsense libraries

### ISSUE THREAD

**Issue by edwinRNDR** [edwinRNDR/librealsense#2]
As noted by @cansik, the current gradle build scripts do not bundle realsense binaries.

**Comment by cansik** [comment]
I would like to work a bit on the library but I am really struggling with gradle and the native build. It is not possible to apply the `java` plugin, so IntelliJ won't be able to detect the source folders and I am not able to add test dependencies (e.g. junit).

Or am I doing something wrong. How do you develop and how have you set it up?

**Comment by edwinRNDR** [comment]
@cansik I agree. The Gradle setup is pretty painful to work with and admittedly much of what `org.bven.jniplugin` introduces is beyond my understanding of Gradle. I think we would benefit from looking for an alternative that allows a conventional build workflow. 

My setup is the same as yours, I load the project in IntelliJ but source folders are not detected and I end up working without most of the tooling.
